controllers/user.controller.js

```diff
@@ -0,0 +1,48 @@
+const User = require("../models/user.model");
+
+exports.create = function(req, res){
+    let user = new User(
+        {
+            username: req.body.username,
+            password: req.body.password
+        }
+    );
+
+    user.save(function (err) {
```

Comment:
we will need to set up some express middleware to handle errors, so you don't need to explicitly say ```res.send("Error")```

Reply:
But for now, this looks fine